src/main/java/core/basesyntax/RightTriangleSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangleSupplier {

```

Comment:
This class does not have method to generate a default white circle with a radius of 10 as per the task requirement.

Reply:
It is in FiugureSupplier. I guess that was the instruction.